Cohort2/team4/client/src/pages/events/index.tsx

```diff
@@ -4,23 +4,51 @@ export default function EventDashboard() {
   const user = "Sally";
   const events = ["event1"];
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <div>
-        <header className="flex items-center gap-[70vw]">
-          Convi
-          <button className="bg-[#a6c1ee] text-white px-5 py-2 rounded-full hover:bg-[#87acec]">
-            Sign in
-          </button>
-        </header>
-      </div>
-      <div>
-        <div className="space-y-10">
+    <main style={{ height: "100dvh" }}>
+      <div className="grid grid-row-1 gap-12 my-20 mx-auto w-3/5 h-2/3 ">
+        <div className="bg-[#f1f5f9] space-y-10 border-grey border-solid border-2 pl-20 pt-20 rounded-md">
           <h1 className="font-bold text-2xl">Hello there, {user}</h1>
+          <div className="grid grid-cols-2">
           <p>See your upcoming events</p>
-          <EventList events={events} />
-          <button className="bg-[#a6c1ee] text-white px-5 py-2 rounded-full hover:bg-[#87acec] m-10">
-            Add New Event
-          </button>
+          <p className="">Days</p>
+          </div>
+          <div className="grid grid-cols-2 justify-between">
+            <form action="" className="flex flex-col space-y-4">
```

Comment:
These don't need to be forms. Ideally they wouldn't be. We won't be doing data entry on them, just onClick actions. I'd remove and replace with a different container.